.github/workflows/announce-calls.yml

```diff
@@ -0,0 +1,23 @@
+name: Announce calls
+on:
+  schedule:
+    # 8th to the 14th of each month at 9am.
+    - cron: '0 9 8-14 * *'
+    # Thusdays at 9am.
+    - cron: '0 9 * * 4'
```

Comment:
Should this be two hours earlier to allow for the same amount of notice for both calls?

Reply:
Yea that makes sense! Will adjust...

Reply:
Force pushed.